### ISSUE TITLE
Implement "phone_call" model

### ISSUE THREAD

**Issue by dominiqueemmanuel** [ropensci/googleLanguageR#46]
Hi,

Thanks again for your work in this package!

I am wondering if it is possible to use the (new?) phone_call model from de the Google API? This model is supposed to perform well on audio coming from phone call recording...

Here is the link in the documentation:
[https://cloud.google.com/speech-to-text/docs/phone-model](https://cloud.google.com/speech-to-text/docs/phone-model)

Best regards,
Dom

**Comment by MarkEdmondson1234** [comment]
Good question - this model is not yet in as it first needs data logging options enabled ( https://cloud.google.com/speech-to-text/docs/data-logging ) but then will add phone_call and video models.  As its a beta service I try to keep those features on GitHub only once developed in case the API changes, but when it is fully released will merge into the CRAN release.

**Comment by raroito87** [comment]
Hi,

I am getting an error when I try to use the phone_call model with "de-DE". I enabled data logging and set useEnhanced=True. I imagine the phone call model is not yet compatible with german language? or am I missing something else?

**Comment by MarkEdmondson1234** [comment]
@raroito87 You can see the language support here: https://cloud.google.com/speech-to-text/docs/languages

Looks like German is not supported for the phone model as you say
